### DISCUSSION TITLE
Ошибка компиляции 'workMode' was not declared in this scope

### DISCUSSION THREAD

**Discussion by Robot-cube** [vvip-68/GyverPanelWiFi#291] [Задавайте вопросы] [answered]
При компиляции выходит несколько предупреждений и ошибка. Почему-то жалуется на языки и в конце вылетает с ошибкой 'workMode' was not declared in this scope
Подскажите, в какую сторону копать. Пробовал несколько раз, и библиотеки копировал, и среду переустанавливал, в конце вообще удалил Ардуино и поставил чистую с нуля, библиотеки, которые были встроены, перед перезаписыванием - удалял и копировал из проекта.
Плата NodeMCU, пробовал менять настройки, не помогло.
![image](https://github.com/vvip-68/GyverPanelWiFi/assets/154126355/2f038771-ec3e-4835-84f1-8a3aa52c8a7f)

**Comment by vvip-68** [comment, marked as answer]
Это всё странно... Вроде как дня три назад уже всё было исправлено...
Уберите тогда вручную кусок && workMode == SLAVE

**Reply by Robot-cube** [reply]
Оно закомпилилось, прошивка прошла, но дальше ничего: монитор порта показывает кашу, плата судя по всему висит (LED pin не горит), точка доступа не поднимается...

**Reply by vvip-68** [reply]
Чтоб монитор порта не показывал кашу - нужно скорость порта установить в соответствии со скоростью вывода из скеичаю. В данном случае это 115200
 Далее без того, что выводится в порт - гадать бессмысленно
